CHANGELOG.md

```diff
@@ -10,14 +10,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.11.0[[v0.11.0_log]
+--------------
+
 ### Added
-- OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+- Open Enclave SDK Release packages can now be built on non-SGX and non-FLC machines.
 - Support for arbitrarily large thread-local data for SGX machines.
+- Experimental support for OpenSSL inside enclaves has been added while building the SDK from source.
+     - Use BUILD_OPENSSL flag while compiling the SDK
+     - [OpenSSLSupport.md](docs/OpenSSLSupport.md) documents supported options and configuration needed to OpenSSL inside an enclave.
 
 ### Changed
 - Rename the custom claims buffer added by oe_get_evidence from "custom_claims" to "custom_claims_buffer". Likewise, replace the `OE_CLAIM_CUSTOM_CLAIMS` definition for this name with `OE_CLAIM_CUSTOM_CLAIMS_BUFFER`.
+- Building SDK from source		
+      - HAS_QUOTE_PROVIDER cmake option has been removed. This is a continutation of the work in previous release to allo the same build of OE SDK to run on both FLC and non FLC machines."
+      - Intel SGX EnclaveCommonAPI pacakges are no longer needed to build the SDK.
 
```

Comment:
Please add these to "Changed" list:

- `oe_verify_attestation_certificate_with_evidence()` can verify certificates generated by `oe_generate_attestation_certificate()` as well as `oe_get_attestation_certificate_with_evidence()`.
- SGX attestation evidence internal structure changed. Current structure (version 3) is not compatible with previous version. Applications that call `oe_get_evidence()` or `oe_verify_evidence()` have to be rebuilt.
- Some SGX attestation format IDs are renamed:

| Old | New |
| -- | - |
OE_FORMAT_UUID_SGX_ECDSA_P256  | OE_FORMAT_UUID_SGX_ECDSA |
OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT | OE_FORMAT_UUID_LEGACY_REPORT_REMOTE |
OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE | OE_FORMAT_UUID_RAW_SGX_QUOTE_ECDSA |